packages/ember-metal/lib/expand_properties.js

```diff
@@ -38,7 +38,8 @@ var END_WITH_EACH_REGEX = /\.@each$/;
 export default function expandProperties(pattern, callback) {
   assert('A computed property key must be a string', typeof pattern === 'string');
   assert(
-    'Brace expanded properties cannot contain spaces, e.g. "user.{firstName, lastName}" should be "user.{firstName,lastName}"',
+    'Dependent keys passed to Ember.computed.or() cannot contain spaces,' +
```

Comment:
This method is used by all computed properties (and observers), saying computed.or here isn't necessarily correct